tmt-app/src/constants/careerMajorList.js

```diff
@@ -6,96 +6,112 @@ export const CAREERS_LIST = [
     name: "Accounting/Finance",
     desc: "Accounting is the process of recording, classifying, and summarizing financial transactions to provide information that is useful in making business decisions. Finance is the study of how individuals and organizations manage their money, including the processes of acquiring and using financial resources over time. Accountants and financial professionals use tools such as financial statements and spreadsheets to analyze and interpret financial data, and they may also be involved in forecasting and budgeting, risk management, and decision-making",
     link: "https://www.zippia.com/accountant-jobs/",
+    key: "account_finance",
   },
   {
     id: 1,
     name: "Art",
     desc: "An art career often refers to a job that involves being creative or producing works of art, often using a specific medium. This can include professionals who produce their own art, like painters and sculptors, as well as jobs that entail working with art or evaluating its value. Art careers encompass a broad field of subjects, from art history to interior design to illustration. Professionals typically choose a specific concentration in college and hone their skills for that particular industry. Many art careers exist in the entertainment industry, but you can also find artistic jobs in industries like marketing, fashion and news reporting",
     link: "https://www.zippia.com/artist-jobs/",
+    key: "art",
   },
   {
     id: 2,
     name: "Consultant",
     desc: "Consulting is the practice of providing expert advice to organizations in order to help them solve problems, improve their performance, and achieve their goals. Consultants may work in a variety of industries and may specialize in areas such as strategy, operations, finance, or technology. Consultants work with clients to understand their needs, conduct research and analysis, and develop recommendations and solutions to address their challenges. Consultants may also be involved in implementing their recommendations and helping clients to achieve their desired outcomes",
     link: "https://www.zippia.com/consultant-jobs/",
+    key: "consultant",
   },
   {
     id: 3,
     name: "Data Science",
     desc: "Data science is the field that deals with the collection, analysis, and interpretation of large datasets. It involves using statistical and computational techniques to extract insights and knowledge from data, and applying this knowledge to solve real-world problems. Data scientists often use tools such as machine learning algorithms and programming languages like Python to analyze and interpret data",
     link: "https://www.zippia.com/data-scientist-jobs/",
+    key: "data_science",
```

Comment:
Can we use kebab-case for these? I believe URLs have a standard to use hyphens over underscores.

@michachan can you verify?

Reply:
Sounds good! I will make the adjustment